src/main.ts

```diff
@@ -284,6 +286,15 @@ export function prettifyExecutionGraphResult(executionGraphResult: Object): void
           ansi.red(task["tests"]["failed"])
         )} ${ansi.bold(ansi.red("failed"))}`
       )
+      core.summary
+      .addTable([task["action_id"], "action:", `${task["passed"] ? ("passed") : ("failed")}`])
+      .addTable([  
+        ["Tests:"], 
+        [`${(task["tests"]["passed"])} passed`], 
+        [`${task["tests"]["skipped"]} skipped`], 
+        [`${task["tests"]["failed"]} failed`]
+      ])
+      .write() 
```

Comment:
I'd say we should only call `.write()` on the last `summary` usage. Otherwise, by looking at [this](https://github.com/vmware-labs/vmware-image-builder-action/actions/runs/4005629924) execution, it seems it will duplicate tables. I'd try by calling it only right after the `.addRaw...` call.